tools/android/install_base.sh

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env bash
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2024, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Forked from https://github.com/ARM-software/lisa/blob/main/install_base.sh
+#
+
+# shellcheck disable=SC2317
```

Comment:
If that's the only violation found by shellcheck I'm glad :) (although I don't know how deep shellcheck actually looks)

Reply:
Yes, that was the only shellcheck violation at least when I ran it for the last time.